packages/runtime-core/src/componentProps.ts

```diff
@@ -662,7 +663,7 @@ function validateProps(
       resolvedValues[key],
       opt,
       __DEV__ ? shallowReadonly(resolvedValues) : resolvedValues,
-      !hasOwn(rawProps, key) && !hasOwn(rawProps, hyphenate(key)),
+      !hasOwn(rawProps, key) && !camelizePropsKey.includes(key),
```

Comment:
You're right, I'm writing this because I'm not sure if the performance of `includes` will be worse than `hasOwn`, so I only do the following operations when `hasOwn (rawProps, key)` is not met

Reply:
So do you think I should remove the previous judgment

Reply:
I think it is fine because this check only happens in DEV.

Reply:
ok.  I'll fix it

Reply:
@edison1105 done